packages/components/src/PageHeader/PageHeader.stories.tsx

```diff
@@ -633,9 +631,27 @@ export const Subheader = () => (
   </Wrapper>
 );
 
-/** Пример использования  */
-export const Usage = () => (
+export const TooltipSecondaryAction = () => (
```

Comment:
```suggestion
/**
 * Для отображения Tooltip на заблокированном элементе необходимо использовать prop ```disableReason``` и ```disabled```.<br>
 * Так же можно выводить Tooltip на не заблокированном элементе с помощью prop ```note```
 */
export const TooltipSecondaryAction = () => (
```